apps/src/applab/designElements/PropertyRow.jsx

```diff
@@ -15,6 +17,10 @@ var PropertyRow = React.createClass({
     };
   },
 
+  componentWillReceiveProps: function (newProps) {
```

Comment:
I am imagining it will happen when we drag an element in the playspace and then call setProps() on the <Element>Properties component so that the numbers here update, and this seems like this right thing to do in this case. but I am wondering if this method is already getting hit in this PR via a different codepath.